docs/syntax-reference.rst

```diff
@@ -0,0 +1,647 @@
+| Property     | Description                                           |
++==============+=======================================================+
+| KAFKA_TOPIC  | The name of the Kafka topic that backs this table. If |
+|              | this property is not set, then the name of the table  |
+|              | will be used as default.                              |
++--------------+-------------------------------------------------------+
+| VALUE_FORMAT | Specifies the serialization format of the message     |
+|              | value in the topic. Supported formats: ``JSON``,      |
+|              | ``DELIMITED``. If this property is not set, then the  |
+|              | format of the input stream/table will be used.        |
++--------------+-------------------------------------------------------+
+| PARTITIONS   | The number of partitions in the topic. If this        |
+|              | property is not set, then the number of partitions of |
+|              | the input stream/table will be used.                  |
++--------------+-------------------------------------------------------+
+| REPLICATIONS | The replication factor for the topic. If this         |
+|              | property is not set, then the number of replicas of   |
+|              | the input stream/table will be used.                  |
++--------------+-------------------------------------------------------+
+| TIMESTAMP    | Associates the message timestamp in the Kafka topic   |
+|              | with a column in the KSQL table. Time-based           |
+|              | operations such as windowing will process a record    |
+|              | according to this timestamp.                          |
++--------------+-------------------------------------------------------+
+
+DROP STREAM
+-----------
+
+**Synopsis**
+
+.. code:: sql
+
+    DROP STREAM stream_name;
+
+**Description**
+
+Drops an existing stream.
+
+DROP TABLE
+----------
+
+**Synopsis**
+
+.. code:: sql
+
+    DROP TABLE table_name;
+
+**Description**
+
+Drops an existing table.
+
+SELECT
+------
+
+**Synopsis**
+
+.. code:: sql
+
+    SELECT select_expr [, ...]
+      FROM from_item [, ...]
+      [ WINDOW window_expression ]
+      [ WHERE condition ]
+      [ GROUP BY grouping_expression ]
+      [ HAVING having_expression ];
+
+**Description**
+
+Selects rows from a KSQL stream or table. The result of this statement
+will not be persisted in a Kafka topic and will only be printed out in
+the console. To stop the continuous query in the CLI press ``Ctrl-C``.
+
+In the above statements from_item is one of the following:
+
+-  ``stream_name [ [ AS ] alias]``
+-  ``table_name [ [ AS ] alias]``
+-  ``from_item LEFT JOIN from_item ON join_condition``
+
+The WHERE clause can refer to any column defined for a stream or table, including the two implicit columns `ROWTIME`
+and `ROWKEY`.
+
+Example:
+
+.. code:: sql
+    SELECT * FROM pageviews
```

Comment:
this needs a newline between the `.. code` command and its contents. currently it isn't rendering.